src/renderer/port-forward/port-forward.store.ts

```diff
@@ -166,9 +164,10 @@ export async function removePortForward(portForward: ForwardedPort) {
   portForwardStore.reset();
 }
 
-export async function getPortForwards(): Promise<ForwardedPort[]> {
+export async function getPortForwards(clusterId?: string): Promise<ForwardedPort[]> {
   try {
-    const response = await apiBase.get<PortForwardsResult>(`/pods/port-forwards`);
+    const query = clusterId ? `?clusterId=${clusterId}` : "";
```

Comment:
I would pass in the query as the second argument to `apiBase.get()`

Reply:
ah, probably should've been done for the other `apiBase` calls too?

Reply:
Probably